ansible/group_vars/codex.test.yml

```
@@ -1,6 +1,7 @@
 ---
 # Geth
-geth_account_pass: 'verysafe' # FIXME
+geth_account_pass: "{{ 'verysafe' | repeat((poa_public_keys | length) + (eth_public_keys | length)) | join('\n') }}" # FIXME
```

You don't need a filer, we can use a Jinja for loop in a variable:
```suggestion
geth_public_keys_count: '{{ (poa_public_keys|length) + (eth_public_keys|length) }}'
geth_account_pass: |-
  {% for _ in range(0, (geth_public_keys_count|int)) %}
  verysafe
  {% endfor %}
```